src/main/java/seedu/pill/command/FindCommand.java

```diff
@@ -12,10 +14,14 @@ public FindCommand(String itemName) {
     }
 
     @Override
-    public void execute(ItemMap itemMap, Storage storage) {
+    public void execute(ItemMap itemMap, Storage storage) throws PillException {
         ItemMap foundItems = itemMap.findItem(itemName);
-        ListCommand listCommand = new ListCommand();
-        listCommand.execute(foundItems, storage);
+        if (foundItems.isEmpty()) {
+            throw new PillException(ExceptionMessages.ITEM_NOT_FOUND_ERROR);
+        } else {
```

Comment:
Nice work throwing a descriptive item not found message in place of the empty inventory message